app/src/main/java/com/example/findu/presentation/ui/base/FindUButton.kt

```diff
@@ -0,0 +1,54 @@
+package com.example.findu.presentation.ui.base
+
+import androidx.annotation.StringRes
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.example.findu.ui.theme.FindUTheme
+
+@Composable
+fun FindUButton(
+    modifier: Modifier = Modifier,
+    @StringRes textRes: Int,
+    onClick: () -> Unit,
+    enabled: Boolean = true,
+) {
```

Comment:
필수는 아니지만 구글에서 인자 순서를 
1. 필수인자
2. modifier
3. optional인자 
순서를 권장한다고 알고있습니다!
```suggestion
fun FindUButton(
    @StringRes textRes: Int,
    onClick: () -> Unit,
    modifier: Modifier = Modifier,
    enabled: Boolean = true,
) {
```

Reply:
아하 그런 규칙이 있는 줄 몰랐습니다.
https://android.googlesource.com/platform/frameworks/support/+/androidx-main/compose/docs/compose-api-guidelines.md
요 문서에 나와있나요 ??
다시 한 번 공부해야겠습니다..